packages/widgets/src/components/AddNewAccountCard/AddNewAccountCard.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useTheme } from '@emotion/react';
+import { Button, Icon } from '@ssa-ui-kit/core';
+import { AddNewAccountCardProps } from './types';
+import { AccountCard, ResetBtnStyles } from './styles';
+
+export const AddNewAccountCard = ({
+  link,
+  onclick,
+}: AddNewAccountCardProps) => {
+  const theme = useTheme();
+
+  return (
+    <React.Fragment>
+      {typeof onclick === 'function' ? (
+        <Button onClick={onclick} css={[ResetBtnStyles, AccountCard]}>
+          <Icon name="plus" size={12} color={theme.colors.blueRoyal} />
+          Add new account
```

Comment:
Let's not hardcode the text (here and below). Pass it via props.

Reply:
Done